spec/onc_certification_g10_test_kit/bulk_data_group_export_stu2_spec.rb

```diff
@@ -34,7 +34,7 @@ def run(runnable, inputs = {})
   describe '[Bulk Data Server supports "_outputFormat" query parameter] test' do
     let(:runnable) { group.tests.last }
     let(:long_format_req) do
-      stub_request(:get, "#{export_url}?_outputFormat=application/fhir+ndjson")
+      stub_request(:get, "#{export_url}?_outputFormat=application/fhir%2Bndjson")
```

Comment:
should `/` be URL encoded too?

Reply:
(same comment applies to all of these `stub_requests`)

Reply:
Good catch. I'm assuming stubbing doesn't care if its encoded or not. Just changed and pushed up.